docs/UserGuide.adoc

```diff
@@ -165,16 +165,24 @@ Displays the list of spending tagged with `groceries` in `descending order by da
 
 === Find by keyword : `find`
 
-Searches for spending based on given keywords, cost range and date range. +
+Searches for spending based on given keywords, cost range, date range, remarks or tags. +
 At least one search field must be present. +
-Format: `find [n/NAME_KEYWORDS] [c/COST_MIN-COST_MAX] [d/DATE_START - DATE_END] [r/REMARK_KEYWORDS] [t/TAG]...`
+When searching for multiple tags, any tag listed will be matched. +
+Format: `find [n/NAME_KEYWORDS] [c/COST_MIN-COST_MAX] [d/DATE_START] [d/DATE_END] [r/REMARK_KEYWORDS] [t/TAG]...`
 
 Example:
 
-* `find n/Java book c/100.20-150 r/01/09/2019 - 30/09/2019` +
-Returns a list of spending with `Java` and `book` keywords within the cost range `100.20-150` and date range within `01/09/2019 - 30/09/2019`.
-* `find n/apple c/1.50-2.00 r/01/09/2019 to 30/09/2019` +
-Returns a list of spending with `apple` keyword within the cost range `1.50-2.00` and date range within `01/09/2019 to 30/09/2019`.
+* `find n/apple c/1.50-2.00 d/01/09/2019 d/30/09/2019` +
```

Comment:
The different formats of the dates:
`d/ d/`
`d/ .. to...`
`d/ ... - ...`
can be explained in a tip legend.